app/controllers/api/comments_controller.rb

```diff
@@ -0,0 +1,26 @@
+class Api::CommentsController < ApplicationController
+  def index
+    user = User.find(params[:user_id])
+    post = user.posts.find(params[:post_id])
+    comments = post.comments
+    render json: comments.to_json(only: %i[id text created_at updated_at comments_counter likes_counter])
+  end
+
+  def create
+    post = Post.find(params[:post_id])
+    comment = post.comments.build(comment_params)
+    comment.author_id = current_user.id
+
```

Comment:
- [ ] After solving the _**Cross-Origin Request Forgery (CSRF)**_ issue, I noticed I couldn't make a _**POST**_ request successfully. I got this error in the terminal:

![Screenshot_2](https://github.com/skabeo/blog-app/assets/77106962/43f69cd0-2c6d-4d47-9c72-f5589eb1a0c4)

#### Currently, you are seeing the `comment.author_id` to the current_user.id and there is no current_id for this controller because `devise` was not set up for this **_User controller (API controller)_** but the **_web User controller_**. I would advise you to get the User ID the same way you did you did in the index action of this controller. 👍 😎